pkgs/tools/security/yubihsm-shell/default.nix

```diff
@@ -0,0 +1,42 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, openssl
+, libusb
+, libedit
+, curl
+, gengetopt
+, pkg-config
+, pcsclite
+, help2man
+}:
+
+stdenv.mkDerivation rec {
+  pname = "yubihsm-shell";
+  version = "2.3.1";
+
+  src = fetchFromGitHub {
+    owner = "Yubico";
+    repo = "yubihsm-shell";
+    rev = version;
+    sha256 = "sha256-D0kXiwc+i6mKA4oHuHjgXUmLMsmY5o/VI+1aCWtNjC0=";
+  };
+
+  nativeBuildInputs = [ pkg-config cmake openssl help2man ];
```

Comment:
Why is openssl in nativeBuildInputs?

Reply:
I saw openssl in a bunch of nativeBuildInputs elsewhere in nixpkgs, I suppose I didn't think too critically in my copy-pasting efforts! Will fix this now.

Reply:
Fixed [048eb80](https://github.com/NixOS/nixpkgs/pull/169645/commits/048eb80fc1f4fb143dd9efe00f791306e1973645)